build.gradle

```diff
@@ -129,6 +130,12 @@ dependencies {
     compile group: 'com.microsoft.azure', name: 'applicationinsights-core', version: '1.0.9'
     compile group: 'com.microsoft.azure', name: 'applicationinsights-logging-log4j2', version: '1.0.9'
 
+    compile "org.glassfish.tyrus.bundles:tyrus-standalone-client:1.13.1"
+    compile "org.glassfish.tyrus.ext:tyrus-extension-deflate:1.13.1"
+
+    compile "com.google.code.gson:gson:2.8.0"
```

Comment:
Don't we have another JSON library in JabRef? Maybe via unirest-java?

Reply:
We already have Guava. And gson is a much better json library than the default implementation